spec/nostra/account/csv_data_parser_spec.rb

```diff
@@ -0,0 +1,38 @@
+require_relative '../../../app/nostra.rb'
+
+RSpec.describe Nostra::CSVDataParser do
+  # NOTE: currently using sample from actual 1Money export
+  let(:subject) { Nostra::CSVDataParser.transactions('spec/fixture/1Money-sample.html') }
+
+  it 'returns an array of hashes' do
+    expect(subject).to be_a(Array)
+    expect(subject.count).to eql(9)
+    expect(subject.first).to be_a(Hash)
+  end
+
+  it 'has a date' do
+    expect(subject[0][:date]).to eql("8/3/19")
```

Comment:
I'd probably use a `let` to name `subject[0]`.  Also, its making me think that these tests of the objects in the collection might belong in another module?  Like the data parser parses the whole collection, but there's a row parser that does this part?